### DISCUSSION TITLE
An error about GUI

### DISCUSSION THREAD

**Discussion by CaoChengwei9** [The-OpenROAD-Project/OpenROAD#2490] [Q&A]
Hi, when I enter the command `openroad -gui`  to check the results of [OpenFASOC](https://github.com/idea-fasoc/OpenFASOC), I meet this error:
`[ERROR] This code was compiled with the GUI disabled.  Please recompile with Qt5 if you want the GUI.`
`%` 
SO, how to rebuild openroad with Qt5? Thanks.

**Comment by maliberty** [comment]
When you built OR couldn't find Qt installed so it compiles without it.  Please make sure you have Qt installed on your system.  The easiest way it to use https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/etc/DependencyInstaller.sh to get all needed dependencies.
